src/repositories/jobRepository.ts

```diff
@@ -35,7 +41,7 @@ export class JobRepository extends BaseRepository {
       update,
       `Mongo Timeout Error: Timed out while updating success status for jobId: ${id}`
     );
-    if (bRet) {
+    if (bRet && shouldNotifySqs) {
```

Comment:
Added to prevent needing to send out a new SQS message when we can handle calling any post-build functions directly in the new webhook implementation with custom fields. Feel free to let me know if there's a preference to rely on an SQS message here!